src/marina_pool.erl

```diff
@@ -208,11 +211,12 @@ stop_nodes(NodesToStop, Strategy, NewNodes) ->
     [
         begin
             NodeId = node_id(RpcAddress),
-            ets:delete(?MODULE, NodeId),
+            ets:insert(?MODULE, {NodeId, true}),
```

Comment:
Why stop_nodes calls ets:insert and start_node calls ets:delete? And the data structures inserted and deleted don't match..

Reply:
this ets table contains the *dead* node, not the running node.